### DISCUSSION TITLE
Does gismo include examples about gap or overlap?

### DISCUSSION THREAD

**Discussion by BigMaxBig** [gismo/gismo#580] [Q&A]
Since I didn't find examples in your paper, which is named 'Discontinuous Galerkin Isogeometric Analysis on Non-matching Segmentation: Error Estimates and Efficient Solvers', may I ask for one of these examples for learning?

**Comment by BigMaxBig** [comment]
Looking forward to your reply. If it is no, please just give me a response. Thanks a lot.

**Comment by dominik-mokris-mtu** [comment]
As far as I can see, those examples are not in the publically available part of Gismo. If you still want them, I would recommend contacting the authors directly per e-mail.

Or maybe we can shift them to stable? @filiatra, @stefantakacs ?
